pprofile.py

```diff
@@ -883,14 +962,37 @@ def runpath(path, argv, filename=None, threads=True, verbose=False):
     """
     _run(threads, verbose, 'runpath', filename, path, argv)
 
-_allsep = os.sep + (os.altsep or '')
+def _qCacheGringAdapt(path):
+    '''
+    qCacheGrind (windows build) needs at least one UNIX separator
+    in path to find the file. Adapt here even if this is probably
+    more of a qCacheGrind issue...
+    '''
+
+    if os.path.sep != "/":
```

Comment:
Well, in the case of qcachegrind, this is specifically for the callgrind output, so it made sense to be local to that method.

Also, it is (from the beginning) quite a dirty hack: it uses profiled machine details (directory separator) to guess which callgrind dialect to produce (hence, details on the analysis machine).

Is there a reason to reuse it in other places ? I would prefer to avoid spreading this.